documentation/source/users/rmg/credits.rst

```diff
@@ -4,7 +4,9 @@
 Credits
 *******
  
-RMG is based upon work supported by the Department of Energy, Office of Basic Energy Sciences through grant DE-FG02-98ER14914 and by the National Science Foundation under Grants No. 0312359 and 0535604.
+RMG is based upon work supported by the Department of Energy, Office of Basic Energy Sciences through grant DE-FG02-98ER14914 and  as part of the Computational Chemical Sciences Program. 
+and by the National Science Foundation under Grants No. 0312359, 0535604, and 1751720.
```

Comment:
Weird line change? Should these two be in the same sentence?

Reply:
@rwest Do you know if the second line should be removed, or should they be combined with the first line? I'm not familiar with the RMG funding source situation.